pointers/pointers.c

```diff
@@ -8,7 +8,9 @@
 */
 void swap(int* a, int* b)
 {
-
+  int c = *a;
+  *a = *b;
+  *b = c;
```

Comment:
This is a great start! Pointers are arguably the hardest thing in the Lambda curriculum, and it seems to me that you have a good grasp on the basics.